schema/spacewalk/upgrade/susemanager-schema-4.4.2-to-susemanager-schema-4.4.3/001-rhnKSInstallType.sql

```diff
@@ -0,0 +1,6 @@
+insert into rhnKSInstallType (id, label, name) (
+        select sequence_nextval('rhn_ksinstalltype_id_seq'),
+               'rhel_9','Red Hat Enterprise Linux 9'
+        from dual
+        where not exists (select 1 from rhnKSInstallType where  label = 'rhel_9')
+    );
```

Comment:
Wrong schema migration directory. We are still at susemanager-schema-4.4.1 .
So the migration need to go into a directory with the name 

   susemanager-schema-4.4.1-to-susemanager-schema-4.4.2

I think this is also the reason for the schema migration test errors